armbian/build
# Rockchip-current: Add the GPU 500Mhz OPP back

As stated in the logs :

```
Those were removed from mainline kernels by Chromebook devs,
wanting to protect their broken babies at all cost.

This patch was dully tested here :
https://forum.armbian.com/topic/13515-panfrost-on-rk3288-and-gpu-on-600mhz-problems/

But you're welcome to test it again !

I did it, with a whole glmark2 round and it worked flawlessly.
```

To test the GPU at that frequency, locate the **devfreq** folder of your GPU unit on your system (For tinkerboards, it's in **/sys/devices/platform/ffa30000.gpu/devfreq/ffa30000.gpu**) and then :

* Check the available frequencies by doing `sudo cat available_frequencies` .  
That should output something like this : `100000000 200000000 300000000 400000000 500000000 600000000`  
Be sure that `500000000` is in the list.

* Setup the minimum GPU frequency to 500Mhz like this : `sudo echo 500000000 > min_freq`

You can also set up the GPU to not go over 500Mhz like this : `sudo echo 500000000 > max_freq`
